testing/matrix-sdk-integration-testing/src/tests/e2ee/shared_history.rs

```diff
@@ -127,13 +128,15 @@ async fn test_history_share_on_invite() -> Result<()> {
     info!(room_id = ?alice_room.room_id(), "Alice has created and enabled encryption in the room");
 
     // ... and sends a message
-    alice_room
+    let event_id = alice_room
         .send(RoomMessageEventContent::text_plain("Hello Bob"))
         .await
-        .expect("We should be able to send a message to the room");
+        .expect("We should be able to send a message to the room")
+        .event_id;
 
     // Alice invites Bob to the room
     alice_room.invite_user_by_id(bob.user_id().unwrap()).await?;
+    alice_sync_service.stop().await;
```

Comment:
might be worth a comment to say why this is here?

Reply:
Oh, Alice is basically done and doesn't need to continue syncing, it produces unnecessary logs otherwise.

Reply:
apparently I already said that. I think there's an echo in here.

Reply:
Yes yes, I'll add a comment.

Reply:
could you add a comment please?

Reply:
Oh no, he's stuck in a loop.